Components/Queues/Drivers/PhpAmqpLibDriver.php

```diff
@@ -98,30 +75,32 @@ class PhpAmqpLibDriver extends Service implements PurgeableQueueDriverInterface
      * PhpAmqpLibDriver constructor.
      * @param QueueManager $manager
      */
-    public function __construct(PhpAmqpLibQueueManager $manager)
+    public function __construct(PhpAmqpLibQueueManager $manager, string $format = null)
     {
         parent::__construct();
         $this->manager = $manager;
-        $this->format = self::FORMAT_JSON;
+        $this->format = $format ?? self::FORMAT_JSON;
     }
 
     /**
      * PhpAmqpLibDriver destruct
      */
-    public function __destruct()
+    final function __destruct()
```

Comment:
Which is the reason behind to declare `__destruct()` as final? Also it is missing the visibility of the method (public, protected or private)

Reply:
IDE stuffs, I rolled back to the former.